examples/platform/linux/AppMain.cpp

```diff
@@ -222,8 +222,8 @@ CHIP_ERROR InitCommissioner()
     ReturnErrorOnFailure(gFabricStorage.Initialize(&gServerStorage));
 
     factoryParams.fabricStorage = &gFabricStorage;
-    // use a different listen port for the commissioner.
-    factoryParams.listenPort              = LinuxDeviceOptions::GetInstance().securedCommissionerPort;
+    // use a different listen port for the commissioner than the default used by chip-tool.
+    factoryParams.listenPort              = LinuxDeviceOptions::GetInstance().securedCommissionerPort + 10;
```

Comment:
Why are we listening on a hardcoded port here to start with?  Can we not use an ephemeral port?

Reply:
linuxDeviceOptions can be overridden from the command line so the port choice is already change-able

Reply:
Yes, but why are we not using an ephemeral port when no command-line option is provided?  @andy31415 @chrisdecenzo